customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.plulist.v1.json",
+  "title": "PLU List messages",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "id",
+      "plu"
+    ],
+    "properties": {
+      "effectiveDate": {
+        "description": "The date when this PLU list should be applied, if present.",
+        "type": "string",
+        "format": "date",
+        "example": "2021-01-01"
+      },
+      "id": {
+        "description": "A unique identifier for the PLU list.",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 20,
+        "example": "top-5-items-plu"
+      },
+      "name": {
+        "description": "The PLU List name",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 40,
+        "example": "PLU List"
+      },
+      "description": {
+        "description": "The PLU List description",
+        "type": "string",
+        "minLength": 1,
+        "maxLength": 255,
+        "example": "The description for PLU List"
+      },
+      "deviceTypes": {
```

Comment:
Is this per category or per entire PLU? We need it to be per catergory so that a till can fetch SCO or POS categorie

Reply:
so, in this case, instead of an array of lists we can have a single value, and can we change type at line 5 from array to object?